src/meltano/core/project_init_service.py

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
I'd request a few more changes to this since it's changing an assumption about the init experience:

* Update the "Created / creating" output

```bash
Created .                   # should be removed
Creating project files...
  ./                        # can this be the current working directory? Or at least the current folder.
   |-- .meltano
   |-- meltano.yml
```

* `Project . has been created!` should be `Your project has been created!`

Reply:
https://github.com/meltano/meltano/pull/6814/commits/acf4bd22ce3f7d1c665800d84b02b75783ef7577